src/Documentation/fpga/xrt.rst

```diff
@@ -471,6 +471,27 @@ these subsystems will be handled in leaf drivers with root and partition drivers
 being part of the infrastructure and provide common services for all leaves found
 on all platforms.
 
+The driver object model looks like the following::
+
+                    +-----------+
+                    |   root    |
```

Comment:
maybe we can put source file name here? root (xmgmt-root.c)?

Reply:
Or maybe put a _info_ box on where to look for code for these objects?

Reply:
not in this diagram, but in each related text section?

Reply:
I meant something like a legend which can be done with the note directive
.. note:: 

https://sublime-and-sphinx-guide.readthedocs.io/en/latest/notes_warnings.html

We already use it in our xrt.rst